src/mz-debug/src/utils.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License in the LICENSE file at the
+// root of this repository, or online at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fs::{create_dir_all, remove_dir_all, File};
+use std::io::{copy, BufWriter};
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use chrono::{DateTime, Utc};
+use zip::write::SimpleFileOptions;
+use zip::ZipWriter;
+
+static DEBUG_FOLDER: &str = "mz-debug";
+
+pub fn format_base_path(date_time: DateTime<Utc>) -> PathBuf {
+    let path = PathBuf::from(DEBUG_FOLDER).join(date_time.format("%Y-%m-%dT%H:%MZ").to_string());
```

Comment:
Can we instead just put this directly in the current directory, named with an `mz-debug` prefix?

Why do we need to nest this in that DEBUG_FOLDER?

Reply:
Ah are you referring to all files generated or just the stdout/stderr file? Mainly because I want a centralized location to zip everything such that the user can send it to me!

Reply:
I meant, why have it as `mz-debug/%Y-%m-%dT%H:%MZ` as opposed to something like `mz-debug_%Y-%m-%dT%H:%MZ`? If everything is going inside that dated folder, why have a base directory that just contains the date directory?